.gitlab-ci-prod.yml

```diff
@@ -39,3 +39,14 @@ deploy:
   only:
     - prod
   when: manual
+
+testing:
+  stage: deploy
```

Comment:
Please change `stage` to `test`, and add a `test` stage to `stages` on line 12

see https://github.com/HumanCellAtlas/data-store/blob/7af5da0cd11f158bcf4645c89099df448a24cbe6/.gitlab-ci.yml#L12

Reply:
resolved